.husky/pre-commit

```diff
@@ -1,4 +0,0 @@
-#!/usr/bin/env sh
```

Comment:
You should add pre-commit hook as per module, you can add linting & formatting may be

Reply:
To be addressed in next MR